Python/bytecodes.c

```diff
@@ -2135,10 +2135,8 @@ dummy_func(
             _GUARD_DORV_NO_DICT +
             _STORE_ATTR_INSTANCE_VALUE;
 
-        inst(STORE_ATTR_WITH_HINT, (unused/1, type_version/2, hint/1, value, owner --)) {
+        op(_STORE_ATTR_WITH_HINT, (hint/1, value, owner --)) {
             PyTypeObject *tp = Py_TYPE(owner);
```

Comment:
```suggestion
```

This is only used in the assert, AFAICT, so will generate a compiler warning for non-debug builds.